### PR TITLE
Showing banner again after finish scan management menu

### LINKED ISSUE #805: [FEATURE REQUEST] Showing banner again after finish scan management menu
**Is your feature request related to a problem? Please describe.**
After entering Scan Management Menu, the banner that showing information about target is missing.

**Describe the solution you'd like**
Everytime user exit the scan management menu, it will show banner info and progress bar.

**Additional context**

Before this patch, the program is like this

SCR-20230306-rete-2

If you take screenshot of the output, it's so messy and if you want to crop the result, it's only progress bar without banner. But take a look at this

SCR-20230306-rfkk

The result still can be cropped without scan management menu and resulting nice output like this

SCR-20230306-rfly

So, my concern is clean look when we take screenshot for evidence.

### PR DESCRIPTION
**Pull request by aancw**
Fixes #805 

## Branching checklist
- [x] There is an issue associated with your PR (bug, feature, etc.. if not, create one)
- [x] Your PR description references the associated issue (i.e. fixes #123456)
- [x] Code is in its own branch
- [x] Branch name is related to the PR contents
- [ ] PR targets main

## Static analysis checks
- [x] All rust files are formatted using `cargo fmt`
- [x] All `clippy` checks pass when running `cargo clippy --all-targets --all-features -- -D warnings -A clippy::mutex-atomic`
- [x] All existing tests pass

## Documentation
- [ ] New code is documented using [doc comments](https://doc.rust-lang.org/stable/rust-by-example/meta/doc.html)
- [ ] Documentation about your PR is included in the `docs`, as needed. The docs live in a [separate repository](https://epi052.github.io/feroxbuster-docs/docs/). Update the appropriate pages at the links below.
  - [ ] update [example config file section](https://epi052.github.io/feroxbuster-docs/docs/configuration/ferox-config-toml/)
  - [ ] update [help output section](https://epi052.github.io/feroxbuster-docs/docs/configuration/command-line/)
  - [ ] add an [example](https://epi052.github.io/feroxbuster-docs/docs/examples/)
  - [ ] update [comparison table](https://epi052.github.io/feroxbuster-docs/docs/compare/)

## Additional Tests
- [ ] New code is unit tested
- [ ] New code is integration tested, as needed
- [ ] New tests pass

### CODE REVIEW

**Comment by epi052** [comment]
thanks for taking the time to put this in! I'll be able to review it this evening most likely. Just wanted to let you know I see it, and thanks again!

**Comment by aancw** [comment]
Okay, take your time.

File: src/scan_manager/scan_container.rs [hunk 1]
```diff
@@ -24,7 +25,7 @@ use std::{
     collections::HashSet,
     convert::TryInto,
     fs::File,
-    io::BufReader,
+    io::{stderr, BufReader},
```
Comment (epi052) [inline src/scan_manager/scan_container.rs:28]: can remove, we'll use the menu's terminal object

File: src/scan_manager/scan_container.rs [hunk 2]
```diff
@@ -8,7 +8,8 @@ use crate::filters::{
 use crate::traits::FeroxFilter;
 use crate::Command::AddFilter;
 use crate::{
-    config::OutputLevel,
+    banner::Banner,
+    config::{Configuration, OutputLevel},
```
Comment (epi052) [inline src/scan_manager/scan_container.rs:12]: can remove Configuration, we'll use the one passed around with `handles`

File: src/scan_manager/scan_container.rs [hunk 3]
```diff
@@ -446,6 +447,12 @@ impl FeroxScans {
         };
 
         self.menu.clear_screen();
+
+        let config = Configuration::new().unwrap();
+        let target = &config.target_url;
+        let banner = Banner::new(&[String::from(target)], &config);
+        banner.print_to(stderr(), Arc::new(config)).unwrap();
```
Comment (epi052) [inline src/scan_manager/scan_container.rs:454]: give this a try, but it should work

```suggestion
        let banner = Banner::new(&[handles.config.target_url.clone()], &handles.config);
        banner
            .print_to(&self.menu.term, handles.config.clone())
            .unwrap_or_default();
```

**Comment by epi052** [comment]
hey there! i had a chance to review. do you mind making the suggested changes whenever is convenient? 

Once done, it should be good to merge.

**Comment by aancw** [comment]
Hey, thanks for the suggestion. I will try to resolve the issue as soon as possible.

**Comment by epi052** [comment]
no rush at all

**Comment by aancw** [comment]
Done, resolved.

**Comment by epi052** [comment]
tests passed and all looks good; merging. Thanks again! 

@all-contributors add @aancw for code and infra

**Pull request merged**